traitsui/qt4/key_binding_editor.py

```diff
@@ -57,47 +58,34 @@ def init(self, parent):
         """
         self.control = KeyBindingCtrl(self)
 
-    def update_object(self, event):
-        """Handles the user entering input data in the edit control."""
-        try:
-            self.value = value = key_event_to_name(event)
-            self._binding.text = value
-        except:
-            pass
-
```

Comment:
This is being removed because it is unused correct?

Reply:
Yes - it is never called.